db/schema.rb

```diff
@@ -13,55 +13,153 @@
 ActiveRecord::Schema.define(version: 2021_07_14_120419) do
```

Comment:
You've had all the schema dupes come through!

Reply:
🤦